src/DefaultScatterPlot/DefaultScatterPlot.jsx

```diff
@@ -0,0 +1,37 @@
+import React, {Component, PropTypes} from 'react'
+import {Tabs, Tab} from 'material-ui'
+import ScatterPlot from '../ScatterPlot'
+import DataTable from '../DataTable'
+
+export class DefaultScatterPlot extends Component {
+  static propTypes = {
+    columns: PropTypes.array.isRequired,
+    data: PropTypes.array.isRequired,
+    series: PropTypes.array.isRequired,
+    size: PropTypes.string,
+    title: PropTypes.string.isRequired
+  };
```

Comment:
I updated babel on Friday, so we no longer need semi-colons after static properties.